ApollosProject/apollos-embeds
# chore: (Web) Add "X" (close) or Back button to Search Bar

## 🐛 Issue

[(Web) Add "X" (close) or Back button to Search Bar](https://3.basecamp.com/3926363/buckets/27088350/card_tables/cards/6833413108)

## ✏️ Solution

Get autocomplete instance to close panel from `Searchbar`

## 🔬 To Test

1. Navigate to a searchbar / content with searchbar
2. use the search bar on mobile. When it is fullscreen, there should now be a back arrow to exit
3. Search should work the same otherwise

## 📸 Screenshots


https://github.com/ApollosProject/apollos-embeds/assets/68402088/d2d7d948-2f49-43dc-be6a-3ddd7d0d2921
